src/main/java/com/twilio/oai/HttpMethod.java

```diff
@@ -0,0 +1,19 @@
+package com.twilio.oai;
+
+import java.util.Arrays;
+
+enum HttpMethod {
+    DELETE,
+    GET,
+    PATCH,
+    POST,
+    PUT;
```

Comment:
Should PATCH and PUT be removed since there aren't any operations tagged to it?

Reply:
Having them there now won't hurt anything and it allows for future support.